crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -425,25 +425,6 @@
 //! be a bit tedious though, so views and rooms have an additional way of
 //! subscribing to updates via [`futures_signals`][].
 //!
-//! As already touched on in
```

Comment:
It's a bit unfortunate to just delete these docs, but I think we really need to control the size of the public API, and these things weren't used at all in ffi or jack-in, so making them private and not documenting them seems like the right thing.

Reply:
Yeah, I totally agree on this. Let's add getter if we need to, but for the moment, let's reduce the surface of potential bugs.